### ISSUE TITLE
Game mode - Team

### ISSUE THREAD

**Issue by Zemanzo** [Zemanzo/web-marbles#96]
**Description**
Two teams will be formed, allowing to let players pick either team by adding that team's color after !marble. e.g. !marble red will join the red team. Point pool will only be divided across the winning team.

**Minimum Acceptance Criteria**
- [ ] Game mode will apply for a single round
- [ ] Create two groups that players can enter
- [ ] Entering is based on skin (e.g. !marble red will join the red team)
- [ ] Entering with a skin that belongs to neither team will enter the marble in a random team, and the skin will be overridden with that team's skin
- [ ] The point pool will only be divided across the team that had one of their marbles finish as first

**Is blocked by the following issues**
#21
#137
#296

**Comment by Venlious** [comment]
Sounds like a cool idea! Just to make sure I somewhat understand the point pool; would this mean that if one team has fewer members, yet still manages to finish the race first, they would get a bigger cut of the pool so to say?

**Comment by Zemanzo** [comment]
@Venlious This is correct! All marbles entered still count towards the total point pool (scoring system has yet to be designed, see #21), so less players in a team means more points per player.
